05-Time-Series-Raster.Rmd

```diff
@@ -92,102 +93,119 @@ This lesson is a part of a series of raster data in R lessons:
 * [Lesson 05 - Raster Time Series Data in R]({{ site.baseurl}}/R/Raster-Times-Series-Data-In-R/)
 * [Lesson 06 - Plot Raster Time Series Data in R Using RasterVis and LevelPlot]({{ site.baseurl}}/R/Plot-Raster-Times-Series-Data-In-R/)
 * [Lesson 07- Extract NDVI Summary Values from a Raster Time Series]({{ site.baseurl}}/R/Extract-NDVI-From-Rasters-In-R/)
-</div>
 
-In this lesson, we will use the `raster`, `rgdal` and `rasterVis` libraries.
+###Additional Resources
+* <a href="http://cran.r-project.org/web/packages/raster/raster.pdf" target="_blank">
+Read more about the `raster` package in `R`.</a>
 
-```{r load-libraries }
+</div>
 
-library(raster)
-library(rgdal)
-library(rasterVis)
+#About Raster Time Series Data
+Raster Data Sets can be made up of multiple files or bands.  These could be
+different specra taken at the same time and location that together offer a
+more complete image of the location. Or these could be single bands taken at
+different times of the same location.  Time series data gives us the ability to 
+measure and monitor changes through time.  
 
-```
+<figure>
+    <a href="{{ site.baseurl }}/images/raster_timeseries/GreenessOverTime.png">
+    <img src="{{ site.baseurl }}/images/raster_timeseries/GreenessOverTime.png"></a>
+    <figcaption>A raster dataset can contain a time series. In R, a stack of
+    rasters will be in the same extent, CRS and resolution. Image Source: NEON,
+    Inc. 
+    </figcaption>
+</figure>
 
-#About the Time Series Data
+In this lesson, we are working with a set of rasters that were derived from the 
+Landsat satellite and are stored in GeoTIFF format. Each raster covers part of
+the  <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">NEON Harvard Forest field site</a>.
+
+In this lesson, we will
 
-In this lesson, we are working with a set of rasters, that were derived from the 
-Landsat satellite - in `GeoTiff` format. Each
-raster covers part of the  <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">NEON Harvard Forest field site</a>.
+1. Import NDVI data derived from the Landsat Sensor in raster GeoTIFF format
+2. Plot one full year of NDVI raster time series data. 
+3. Compare RGB imagery with NDVI data to understand strange values in the NDVI. 
 
 ##NDVI data
-The first set of rasters, located in the `Landsat_NDVI\HARV\ndvi` is the Normalized
-Difference Vegetation Index (NDVI). NDVI is a quantitative index of greeness ranging
-from 0-1 where 0 is the least amount of greenness and 1 is maximum greenness following 
-the index. NDVI is often used for a quantative measure of vegetation health, cover
-and vegetation phenology (life cycle stage) over large areas. 
-The NDVI data is a derived single band product saved as a `geotiff` for different
+The rasters that we will work with, located in the source folder
```

Comment:
The rasters we will use in this lesson, located in the (`Landsat_NDVI\HARV\ndvi`) directory, are Normalized Difference Vegetation Index (NDVI) data. NDVI is a quantitative index of greenness ranging from 0-1 where 0 represents minimal or no greenness and 1 represents maximum greenness.